module/zfs/qat.c

```diff
@@ -99,4 +102,7 @@ qat_fini(void)
 	qat_dc_fini();
 }
 
+module_param(zfs_qat_disable, int, 0644);
+MODULE_PARM_DESC(zfs_qat_disable, "Disable QAT acceleration");
```

Comment:
This can be done via the config files, but I agree it would be nice to be able to disable each one (for instance if you had another application that you wanted to use the compression units). I will make the change.